contracts/jobs/libraries/TranscodeJobs.sol

```diff
@@ -245,6 +248,18 @@ library TranscodeJobs {
     }
 
     /*
+     * @dev Compute the personal segment hash of a segment. Hashes the concatentation of the personal hash prefix and the segment hash
+     * @param _streamId Stream identifier
+     * @param _segmentSequenceNumber Segment sequence number in stream
+     * @param _dataHash Segment data hash
+     */
+    function personalSegmentHash(string _streamId, uint256 _segmentSequenceNumber, bytes32 _dataHash) internal constant returns (bytes32) {
```

Comment:
As of TestRPC v4.0.0 the `eth_sign` RPC call (which `web3.eth.sign` uses) will accept an arbitrary message that it concatenates with a pre-defined prefix and then hashes the result. This "personal hash message" is then used in combination with a private key to generate a signature. https://github.com/ethereumjs/testrpc/pull/244 We replicate this behavior here, by generating the segment hash and then hashing the concatenation of the pre-defined prefix `personalHashPrefix` and the segment hash.